contracts/test/DelegateCaller.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.7.0 <0.9.0;
+
+/**
+ * @title DelegateCaller - A test contract that executes delegatecalls
+ */
+contract DelegateCaller {
+    /**
+     * @notice makes a delegatecall
+     * @param _called The address to be delegate called
+     * @param _calldata the calldata of the call
+     */
+    function makeDelegatecal(address _called, bytes memory _calldata) external returns (bool success, bytes memory returnData) {
```

Comment:
could you please fix the typo in the function name?

Reply:
Done, thanks